src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Collections.Immutable.cs" />
+    <Compile Include="System.Collections.Immutable.netstandard21.cs" Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))" />
```

Comment:
Why is this needed, out of curiosity?

Reply:
I don't see any `netstandard2.1` TFMs here, wouldn't it make more sense to just call this `netcoreapp`, per convention in other projects:

https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.csproj#L9

Reply:
Fixed.
Previously because I was struggled by TFM variable "NetCoreAppCurrent", what are the possible values of it ?